pkg/local_object_storage/metabase/status.go

```diff
@@ -146,13 +156,31 @@ func readBuckets(tx *bbolt.Tx, cID cid.ID, objKey []byte) []BucketValue {
 		if b != nil {
 			v := b.Get(objKey)
 			if v != nil {
-				res = append(res, BucketValue{
+				oldIndexes = append(oldIndexes, BucketValue{
 					BucketIndex: lockedPrefix,
 					Value:       bytes.Clone(v),
 				})
 			}
 		}
 	}
 
-	return res
+	mBucket := tx.Bucket(metaBucketKey(cID))
+	if mBucket == nil {
+		return oldIndexes, nil
+	}
+
+	c := mBucket.Cursor()
+	pref := slices.Concat([]byte{metaPrefixIDAttr}, objKey)
+	k, _ := c.Seek(pref)
+	for ; bytes.HasPrefix(k, pref); k, _ = c.Next() {
+		kCut := k[len(pref):]
+		i := bytes.Index(kCut, object.MetaAttributeDelimiter)
+		if i == -1 {
+			continue
+		}
+
+		newIndexes = append(newIndexes, HeaderField{K: string(kCut[:i]), V: kCut[i+len(object.MetaAttributeDelimiter):]})
```

Comment:
`V` goes out the `tx`, it must be cloned

we may also use const `attributeDelimiterLen`